modules/ocf_stats/files/munin/munin.conf

```diff
@@ -92,7 +92,7 @@ html_strategy cgi
 #
 # For those with Nagios, the following might come in handy. In addition,
 # the services must be defined in the Nagios server as well.
-#contact.nagios.command /usr/bin/send_nsca nagios.host.comm -c /etc/nsca.conf
+contact.root.command | /usr/local/bin/mail-munin-alert /usr/local/bin/mail-munin-alert ${var:host} ${var:wfields} ${var:cfields}
```

Comment:
comment above doesn't match the code below, maybe just remove the comments about nagios?